websites/M/MyTF1/presence.ts

```diff
@@ -0,0 +1,165 @@
+const presence = new Presence({
+		clientId: "1043951049800695808",
+	}),
+	strings = presence.getStrings({
+		viewHome: "general.viewHome",
+		viewPage: "general.viewPage",
+		viewProfile: "general.viewProfile",
+		watchingVid: "general.watchingVid",
+		browse: "general.browsing",
+		live: "general.live",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	myTf1 = "https://i.imgur.com/k8jIoSZ.png",
+	tf1 = "https://i.imgur.com/pFgnoUS.png",
+	tfx = "https://i.imgur.com/u5dTkHx.png",
+	tmc = "https://i.imgur.com/62uYw2r.png",
+	lci = "https://i.imgur.com/qrRgNgk.png",
+	tf1Series = "https://i.imgur.com/WOmFIm3.png",
+	live = "https://i.imgur.com/xuvR04J.png",
+	replay = "https://i.imgur.com/OZDAx9n.png",
+}
```

Comment:
Asset names should be in `PascalCase`
For example:
* `myTf1` => `MyTF1`
* `tfx` => `TFX`
* `replay` => `Replay`